packages/cli/CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
 ## `6.32.1`
 
-- BugFix: Updated Imperative version to fix web help issues.   
+- BugFix: Updated Imperative version to fix web help issues.
 - Expanded help text of --data-set-type on create data set command by adding an example of creating PDSE. [#52](https://github.com/zowe/zowe-cli/issues/52)
+- Fixed inconsistencies in punctuation for command descriptions by adding missing periods. [#66](https://github.com/zowe/zowe-cli/issues/66)
```

Comment:
Please move this line to be right beneath the "Recent Changes" header, since v6.32.1 has already been released.